internal/runners/show/show.go

```diff
@@ -165,3 +198,108 @@ func scriptsData(project *projectfile.Project, conditional *constraints.Conditio
 
 	return data, nil
 }
+
+func platformsData(owner, project string, branchID strfmt.UUID) ([]string, error) {
+	remotePlatforms, fail := model.FetchPlatformsForCommit(branchID)
+	if fail != nil {
+		return nil, locale.WrapError(fail.ToError(), "err_show_get_platforms", "Could not get platform details for commit: {{.V0}}", branchID.String())
+	}
+
+	platforms := make([]string, len(remotePlatforms))
+	for i, plat := range remotePlatforms {
+		if plat.Kernel == nil || plat.Kernel.Name == nil {
+			continue
+		}
+		platforms[i] = *plat.Kernel.Name
```

Comment:
Is that really the most appropriate field? I see they have a `DisplayName` field, shouldn't we use that? Also, if we have to construct the name ourselves like you are here please also include the version and architecture, because projects may be targeting multiple versions of eg. Linux.

Reply:
The `DisplayName` field is an amalgamation of OS, kernel, version, and architecture. There is an `operating_system` field with `name` and `version` however for `Linux` the name is `CentOS`. I went with this field as it's consistent with what `state platforms` uses. I've updated this code to use the same method that the platforms runner uses.